styles.css

```diff
@@ -570,15 +570,15 @@ header {
 
 @media only screen and (min-width: 768px) {
   body {
-    width: 1440px;
+    width: 100%;
   }
 
   .main-div {
-    width: 1440px;
+    width: 100%;
   }
 
   .main-div main {
```

Comment:
- [ ] Dear @chiztechnology, I have seen your design from the new branch with the changes, I can see you have applied change for the display greater than `768px` screen, Yet for any screen less than that it is fixed on `375px`. Please address this issue as it makes your design nonresponsive. 

## device less than `768px`
- Here we need to take the whole width of the available screen.
![Capture](https://user-images.githubusercontent.com/69077061/200399113-ebd6c921-640e-488b-b60b-8698a1d22ed6.PNG)


## device greater than `768px` (this one is good)
- Here we need to not exceed the available screen. (there should not be a scroll bar, the page should shrink to the available screen)
![Capture-2](https://user-images.githubusercontent.com/69077061/200398939-49ed52b0-6526-47aa-8bc2-2dbc697d9437.PNG)